Sources/swift-format/Subcommands/Lint.swift

```diff
@@ -40,18 +40,21 @@ extension SwiftFormatCommand {
         var totalStmts = 0
         var totalErrors = 0
         var totalWarnings = 0
+        var totalRefactorings = 0
 
         frontend.processStatistics {
           totalStmts += $1.statements
           totalErrors += $1.errors
           totalWarnings += $1.warnings
+          totalRefactorings += $1.refactorings
         }
 
-        let score = 10.0 - ((5.0 * Double(totalErrors) + Double(totalWarnings)) / Double(totalStmts)) * 10.0
+        let score = 10.0 - ((5.0 * Double(totalErrors) + Double(totalWarnings) + Double(totalRefactorings)) / Double(totalStmts)) * 10.0
```

Comment:
Should we add a multiplier for the refactoring? Maybe a 2.0?

Reply:
Discussed this offline and since indentation is not accounted we'll leave refactoring same as warning for now.